modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/RexImpTable.java

```diff
@@ -376,6 +380,10 @@ public class RexImpTable {
         defineMethod(UNIX_MICROS, "unixMicros", NullPolicy.STRICT);
         defineMethod(DATE_FROM_UNIX_DATE, "dateFromUnixDate", NullPolicy.STRICT);
         defineMethod(UNIX_DATE, "unixDate", NullPolicy.STRICT);
+        defineMethod(DATE, "date", NullPolicy.STRICT);
```

Comment:
We override this call by `map.put(DATE, new CastImplementor());` later in this method, so looks like we don't need this call at all (if everithing works).

Reply:
Seems that `map.put(DATE, new CastImplementor());` isn't needed at all